src/components/CustomerProfile.js

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+
+const CustomerProfile = ({ customerId }) => {
+  const [customerProfile, setCustomerProfile] = useState(null);
+
+  useEffect(() => {
+    if (customerId) {
+      fetch(`https://cyf-react.glitch.me/customers/${customerId}`)
+        .then((response) => response.json())
+        .then((data) => {
+          setCustomerProfile(data);
```

Comment:
Would be good to add a catch here aswell and some error handling similar to the other api requests. Then you can use that to display an error message.